src/Popover/Popover.tsx

```diff
@@ -109,10 +177,12 @@ export default class Popover extends React.Component {
     }
   };
 
-  onClickOutside = (e) => {
+  onClickOutside = (e: MouseEvent) => {
     const childrenNode = ReactDOM.findDOMNode(this.refs.children);
-    const popoverNode = this.isAbsolute() ? this.containerNode : childrenNode.childNodes[1];
-    const el = e.target || e.srcElement;
+    const popoverNode = this.isAbsolute() ? this.containerNode : childrenNode.children[1];
```

Comment:
I've changed every use of `childNodes` to `children` (won't be repeating this comment).

As a reminder: all elements are nodes, but not all nodes are elements.

This is potentially breaking, but we're accessing a lot of properties of `Element`, so this would break if `childNodes` ever returned a children. I think in practice we never access a node that is not an element, so it's more correct now.